conans/test/unittests/client/build/msbuild_test.py

```diff
@@ -145,3 +145,52 @@ def custom_properties_test(self):
                                                                       "MyProp2": "MyValue2"})
         self.assertIn('/p:MyProp1="MyValue1"', command)
         self.assertIn('/p:MyProp2="MyValue2"', command)
+
+    @parameterized.expand([("15", "v141"),
+                           ("14", "v140"),
+                           ("12", "v120"),
+                           ("11", "v110"),
+                           ("10", "v100"),
+                           ("9", "v90"),
+                           ("8", "v80")])
+    def default_toolset_test(self, compiler_version, expected_toolset):
+        settings = MockSettings({"build_type": "Debug",
+                                 "compiler": "Visual Studio",
+                                 "compiler.version": compiler_version,
+                                 "arch": "x86_64"})
+        conanfile = MockConanfile(settings)
+        msbuild = MSBuild(conanfile)
+        command = msbuild.get_command("projecshould_flags_testt_file.sln")
+        self.assertIn('/p:PlatformToolset="%s"' % expected_toolset, command)
+
+    @parameterized.expand([("v141",),
+                           ("v140",),
+                           ("v120",),
+                           ("v110",),
+                           ("v100",),
+                           ("v90",),
+                           ("v80",)])
+    def explicit_toolset_test(self, expected_toolset):
+        settings = MockSettings({"build_type": "Debug",
+                                 "compiler": "Visual Studio",
+                                 "compiler.version": "15",
+                                 "arch": "x86_64"})
+        conanfile = MockConanfile(settings)
+        msbuild = MSBuild(conanfile)
+        command = msbuild.get_command("projecshould_flags_testt_file.sln", toolset=expected_toolset)
```

Comment:
Is there any reason for misspelling it?

Reply:
fixed spelling, it was copy-paste from another test which had spelling error